apps/yaft/util.h

```diff
@@ -404,3 +404,17 @@ sum(struct parm_t* parm) {
 
   return sum;
 }
+
+static inline bool
+is_landscape() {
```

Comment:
Rename to `is_pogo_connected` it might determine the landscape status, but later we could add another source.